### DISCUSSION TITLE
The alternative to M-RET doesn't work or is poorly described

### DISCUSSION THREAD

**Discussion by davemq** [minad/vertico#445] [Q&A] [answered]
I'm trying to understand these alternatives to M-RET mentioned in https://github.com/minad/vertico/blob/main/README.org:

> As an alternative to pressing M-RET, move the selection first to the input prompt and then press RET.

and

> Non-existing candidates can be submitted with M-RET or by moving the point to the prompt.

I thought that might mean move to the beginning of the selection shown in the prompt by typing C-a, then RET, but that just selects the current selection.

In my particular case I want to use a file or buffer or Org Roam node that is non-existent and a subset of an existing item.

**Comment by minad** [comment, marked as answer]
It is hopefully more clear now: https://github.com/minad/vertico/commit/c7ed4310a25114344a2efc90cfb031fc35f26a29

**Reply by minad** [reply]
A typical example is C-x C-f. There you can move up/down between the files in the directory, and also to the input prompt.
